platform-guide.md

```diff
@@ -88,3 +89,47 @@ You may need Gmail API credentials e.g. for testing against production server.
 1. Click `Create credentials` and then select `OAuth client ID`.
 1. Choose `Other`, give it a name, e.g. `teammates` and click `Create`. You will then get shown your client ID details, click `OK`.
 1. Click the `Download JSON` icon.
+
+## Datastore backup and recovery
+
+Google Cloud Datastore provides [managed export/import](https://cloud.google.com/datastore/docs/export-import-entities) as a means of backup and recovery.
+
+### Backup
+
+The backup has been configured to run weekly in the system by setting `app.enable.datastore.backup=true` in `src/main/resources/build.properties`.
+However, a few prerequisites as described [here](https://cloud.google.com/datastore/docs/schedule-export#before_you_begin) need to be done first.
+
+If you wish to do backup outside the scheduled period, you can simply trigger the backup cron job manually from the GCP console.
+
+### Recovery
+
+In order to import a snapshot into the Datastore:
+
+1. Ensure that all the prerequisites described [here](https://cloud.google.com/datastore/docs/export-import-entities#before_you_begin) are done.
+1. Authenticate and set your project ID (e.g. if your project ID is `teammates-john`):
+
+   ```sh
+   gcloud auth login
+   gcloud config set project teammates-john
+   ```
+
+1. Locate the backup job which you want to use. There are two ways to do this:
```

Comment:
Actually we can just let  the admin go to the Storage file explorer to view the name.

![image](https://user-images.githubusercontent.com/13977235/49326326-8545bf00-f58b-11e8-9113-c1cb4987a194.png)

The can remember the name (suppose it is `2018-12-01T08:38:48.280Z`, denote it as X) and then just try:

```
gcloud datastore import gs://teammates-xiaopu.appspot.com/datastore-backups/X/X.overall_export_metadata
```

Reply:
I guess that works, too.